### ISSUE TITLE
[🐛] node:internal/process/promises:279 triggerUncaughtException

### ISSUE THREAD

**Issue by k-cabral** [juice-shop/juice-shop#1874]
# :bug: Bug report

## Description
Tried and failed to run Juice Shop in three different ways: using Docker on host, locally in a VM and using Docker in a VM. Similar to #1380 but not quite the same, since the program quits after the error, making it impossible to apply the same solution as suggested there.

### Is this a regression?

Yes, works* in v13.0.0 but not in v13.3.0 and beyond.

*By "works" I mean to say the proposed solution to #1380, i.e. waiting a while, works. Possibly simply because the error is not fatal in that version, since it still seems to happen.

## :microscope: Minimal Reproduction

1. Create a VM with 1024MB of memory with VirtualBox
2. Install Ubuntu Server 22.04 selecting the Docker snap in the setup
3. Run `sudo docker run --rm -it bkimminich/juice-shop`
(Error also found running locally, i.e. `git clone`, `npm install` and `npm run start`)

I can also provide a copy of a VirtualBox VM where the error occurs, if needed.

## :fire: Exception or Error

On latest:
<pre><code>
info: All dependencies in ./package.json are satisfied (OK)
info: Chatbot training data botDefaultTrainingData.json validated (OK)
info: Detected Node.js version v16.15.1 (OK)
info: Detected OS linux (OK)
info: Detected CPU x64 (OK)
info: Configuration default validated (OK)
info: Entity models 19 of 19 are initialized (OK)
info: Required file server.js is present (OK)
info: Required file index.html is present (OK)
info: Required file tutorial.js is present (OK)
info: Required file styles.css is present (OK)
info: Required file main.js is present (OK)
info: Required file runtime.js is present (OK)
info: Required file polyfills.js is present (OK)
info: Required file vendor.js is present (OK)
info: Port 3000 is available (OK)
(node:1) [DEP0152] DeprecationWarning: Custom PerformanceEntry accessors are deprecated. Please use the detail property.
(Use `node --trace-deprecation ...` to show where the warning was created)
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error
    at Database.<anonymous> (/juice-shop/node_modules/sequelize/lib/dialects/sqlite/query.js:179:27)
    at /juice-shop/node_modules/sequelize/lib/dialects/sqlite/query.js:177:50
    at new Promise (<anonymous>)
    at Query.run (/juice-shop/node_modules/sequelize/lib/dialects/sqlite/query.js:177:12)
    at /juice-shop/node_modules/sequelize/lib/sequelize.js:311:28
    at async SQLiteQueryInterface.rawSelect (/juice-shop/node_modules/sequelize/lib/dialects/abstract/query-interface.js:434:18)
    at async Function.aggregate (/juice-shop/node_modules/sequelize/lib/model.js:1265:19)
    at async Function.sum (/juice-shop/node_modules/sequelize/lib/model.js:1320:12) {
  name: 'SequelizeDatabaseError',
  parent: [Error: SQLITE_ERROR: no such table: Wallets] {
    errno: 1,
    code: 'SQLITE_ERROR',
    sql: 'SELECT sum(`balance`) AS `sum` FROM `Wallets` AS `Wallet`;'
  },
  original: [Error: SQLITE_ERROR: no such table: Wallets] {
    errno: 1,
    code: 'SQLITE_ERROR',
    sql: 'SELECT sum(`balance`) AS `sum` FROM `Wallets` AS `Wallet`;'
  },
  sql: 'SELECT sum(`balance`) AS `sum` FROM `Wallets` AS `Wallet`;',
  parameters: {}
}
</code></pre>

On v13.0.0:
```
> juice-shop@13.0.0 start /juice-shop
> node build/app

info: All dependencies in ./package.json are satisfied (OK)
info: Chatbot training data botDefaultTrainingData.json validated (OK)
info: Detected Node.js version v12.22.7 (OK)
info: Detected OS linux (OK)
info: Detected CPU x64 (OK)
info: Configuration default validated (OK)
info: Required file server.js is present (OK)
info: Required file index.html is present (OK)
info: Required file styles.css is present (OK)
info: Required file tutorial.js is present (OK)
info: Required file vendor.js is present (OK)
info: Required file polyfills.js is present (OK)
info: Required file main.js is present (OK)
info: Required file runtime.js is present (OK)
info: Port 3000 is available (OK)
(node:18) UnhandledPromiseRejectionWarning: SequelizeDatabaseError: SQLITE_ERROR: no such table: Wallets
    at Query.formatError (/juice-shop/node_modules/sequelize/lib/dialects/sqlite/query.js:403:16)
    at Query._handleQueryResponse (/juice-shop/node_modules/sequelize/lib/dialects/sqlite/query.js:72:18)
    at afterExecute (/juice-shop/node_modules/sequelize/lib/dialects/sqlite/query.js:238:27)
    at Statement.errBack (/juice-shop/node_modules/sqlite3/lib/sqlite3.js:14:21)
(node:18) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:18) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
info: Server listening on port 3000
```

## :deciduous_tree: Your Environment

Host:
```
v18.8.0
8.18.0
```

VM:
<pre><code>
v16.17.0
8.15.0
</code></pre>

### Additional Information
Host:
- OS: Arch Linux
- Kernel: `Linux 5.19.3-arch1-1`
- Docker: `Docker version 20.10.17, build 100c70180f`

VM:
- OS: Ubuntu Server 22.04.1 LTS
- Kernel: `Linux 5.15.0-46-generic x86_64`
- Docker: `version 20.10.14, build a224086349`

**Comment by bkimminich** [comment]
Duplicate of #1380 and expected to be fixed in `v14.3.0` thanks to #1892

**Comment by k-cabral** [comment]
I can confirm the issue has been fixed in v14.3.0 :+1:
